### DISCUSSION TITLE
Import koka code from elsewhere on your system?

### DISCUSSION THREAD

**Discussion by chtenb** [koka-lang/koka#439] [Q&A]
Is it possible to import koka code from some other directory elsewhere on your file system? Using the `import` keyword with paths (relative or absolute) does not seem to work, at least on windows. I tried creating a symbolic link, which almost seems to work, but then still absolute paths end up in the kki file, like this:

```
module interface c:/Users/username/prj/kklib/util
```

with the error `prj_kklib_util.kki(1,19): error: invalid syntax`

**Comment by TimWhiting** [comment]
That seems to be an issue with .kki files, but you can provide the `-i` flag to add an "include" directory, and specify paths relative to that as well. To avoid conflicting names you probably want to provide the directory above the directory that has the name of your custom package.

So if you have
```
- mysrc/something
```
with an import
```koka
import util/utils
```
that depends on some utility somewhere else
```
- /somewhere/else/a/package/named/util/utils
```
You can compile the first with:
`koka -e -i/somewhere/else/a/package/named mysrc/something`

You can have multiple include flags. I think you can provide those flags also to the interpreter either when starting the executable or afterwards with `:set -i/...` and you can also provide `compilerArguments` or something to the VSCode settings.

**Reply by chtenb** [reply]
Thanks, that's helpful. Should I create an issue for the kki problem?

**Reply by TimWhiting** [reply]
Yes, I would
